base/changes.txt

```diff
@@ -6,6 +6,13 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2025-02-25  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* ltoutput.dtx (subsection{The output routine configuration components}):
+	Reset various catcodes to their default
+        values in case the page is broken in some unfortunately place
```

Comment:
unfortunately -> unfortunate

Reply:
Agreed, but I am unsure how a place can be "unfortunate": 
Maybe some extra explanation is needed?